lib/parsers/livesplit.rb

```diff
@@ -85,6 +85,7 @@ def v1_6(xml, fast)
         raise "Not a proper LiveSplit run file"
       end
 
+      run[:history].unshift(run[:time]) unless fast || run[:time].in?(run[:history])
```

Comment:
I was just trying to standardize the histories a little bit more so that there isn't a loss of information.